docs/content/integrations/airlift/operator-migration/bash-operator-dbt.mdx

```diff
@@ -0,0 +1,53 @@
+# Operator migration guides: Migrating usage of `BashOperator` for `dbt`
+
+In this page, we'll explain migrating an Airflow `BashOperator` that runs a `dbt` command to Dagster.
+
+### Background
+
+In Airflow, you might have a `BashOperator` that runs a `dbt` command. For example, you might have a task that runs `dbt run` to build your dbt models.
+
+```python file=/integrations/airlift/operator_migration/bash_operator_dbt.py
+from airflow.operators.bash import BashOperator
+
+run_dbt_model = BashOperator(task_id="build_dbt_models", bash_command="dbt run")
+```
+
+### Dagster equivalent
+
+The Dagster equivalent is to instead use the `dagster-dbt` library to run commands against your dbt project. Here would be the equivalent code in Dagster:
+
+```python file=/integrations/airlift/operator_migration/using_dbt_assets.py
+from dagster_dbt import DbtCliResource, DbtProject, dbt_assets
+
+from dagster import AssetExecutionContext
+
+project = DbtProject(project_dir="path/to/dbt_project")
+
+
+@dbt_assets(manifest=project.manifest_path, project=project)
+def my_dbt_assets(context: AssetExecutionContext, dbt: DbtCliResource):
+    yield from dbt.cli(["run"], context=context).stream()
+```
+
+### Migrating the operator
+
+Migrating the operator breaks down into a few steps:
+
+1. Making the dbt project available to both your Airflow and Dagster deployments.
+2. Writing a `@dbt_asset`-decorated function which runs your dbt commands.
+3. Using `dagster-airlift` to proxy execution of the original task to Dagster.
+
+### Step 1: Making the dbt project available & building manifest
+
+First, you'll need to make the dbt project available to the Dagster runtime and build the manifest.
+
+- If you're building your Dagster deployment in a monorepo alongside your dbt and Airflow projects, you can follow this guide: [Monorepo setup](/integrations/dbt/reference#deploying-a-dagster-project-with-a-dbt-project).
+- If you're deploying within a separate repository, you can follow this guide: [Separate repository setup](/integrations/dbt/reference#deploying-a-dbt-project-from-a-separate-git-repository).
+
+### Step 2: Writing a `@dbt_asset`-decorated function
+
+Once your dbt project is available, you can write a function that runs your dbt commands using the <PyObject object="dbt_assets" module="dagster_dbt"/> decorator and <PyObject object="DbtCliResource" module="dagster_dbt"/>. Most dbt CLI commands and flags are supported - to see a full guide on using `dbt_assets`, check out the [dagster-dbt guide](/integrations/dbt/quickstart).
```

Comment:
I wonder if linking the [reference](https://docs.dagster.io/integrations/dbt/reference) instead of the quickstart would make more sense here? Since we mention a "full guide on using `@dbt_assets`.